README.md

```diff
@@ -19,12 +47,11 @@ Logging config can be built into your program's config struct:
 ```go
 import "github.com/mailgun/log"
 
-
 type Config struct {
   // some program-specific configuration
 
   // logging configuration
-  Logging []*log.LogConfig
+  Logging []log.LogConfig
```

Comment:
While you're changing the API, can you rename `LogConfig` to `Config`? https://blog.golang.org/package-names#TOC_3.